checkbox-snap/series_classic20/snap/snapcraft.yaml

```diff
@@ -27,7 +27,7 @@ apps:
     command: bin/odm-certification-app
   configure:
     command: bin/configure
-  service:
+  agent:
     command-chain: [bin/wrapper_local]
     command: bin/checkbox-cli-wrapper service
```

Comment:
This is deprecated, use the new one

Reply:
Sorry, what is deprecated and what is the new one here?

Reply:
Yes, else that bin will always yield a deprecation warning right? or am I missing something?

Reply:
No, you're right, fixed it. Thanks!